strawberry/flask/tests/test_graphqlview.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+import pytest
+
+from flask import Flask, url_for
+from strawberry.flask.views import GraphQLView
+
+
+def init_app(schema=None):
+    app = Flask(__name__)
+    app.debug = True
+    app.add_url_rule(
+        "/graphql", view_func=GraphQLView.as_view("graphql_view", schema=schema)
+    )
+    return app
+
+
+if __name__ == "__main__":
+    test_app = init_app()
+    test_app.run()
```

Comment:
we don't need this for tests, right? :)

Reply:
We need it to create a flask app instance so we can run the tests. As you can see in `line 25` we are calling the test app.